Foundation/NSRunLoop.swift

```diff
@@ -77,7 +77,7 @@ extension NSRunLoop {
     public func runMode(mode: String, beforeDate limitDate: NSDate) -> Bool {
         let runloopResult = CFRunLoopRunInMode(mode._cfObject, limitDate.timeIntervalSinceNow, false)
 #if os(Linux)
-        return runloopResult == 2 || runloopResult == 3
+        return runloopResult == kCFRunLoopRunHandledSource || runloopResult == kCFRunLoopRunTimedOut
```

Comment:
Crap. I remember I fixed that. Probably rebase/merge/revert problem. Sorry, will update.